packages/remark-mark/__tests__/index.js

```diff
@@ -0,0 +1,63 @@
+import dedent from 'dedent'
+import unified from 'unified'
+import reParse from 'remark-parse'
+import remarkStringify from 'remark-stringify'
+import rehypeStringify from 'rehype-stringify'
+import remark2rehype from 'remark-rehype'
+import remarkCustomBlocks from '../../remark-custom-blocks'
+
+import plugin from '../src/'
+
+
+const render = text => unified()
+  .use(reParse, {
+    footnotes: true,
+  })
+  .use(remarkCustomBlocks, {
+    secret: 'spoiler',
+  })
+  .use(plugin)
+  .use(remark2rehype)
+  .use(rehypeStringify)
+  .processSync(text)
+
+const fixture = dedent`
+  Blabla !!ok!! kxcvj !!ok foo!! sdff
+
+  Look !!ok foo!!! straight !!ok foo!!!!! ahead
+
+  This !!should 
+  work!! ok.
+
+  not!! marked!! text!!!
+
+  sdf !!!! df
+
+  sfdgs ! ! dfg !! dgsg ! qs
+
+  With two exclamation marks: \!!marked text!! you'll get !!marked text!!.
```

Comment:
can you also add a test where the escape is at the end; e.g `!!not marked because of escape \!!`

Reply:
good question @vhf ?

Reply:
I don't think it is. No big deal though, I'll merge and fix it before releasing. I'll comment on this PR as soon as it's released.

Reply:
Ok, I'm done. I couldn't find a good way of escaping end marks so let's go without them, to me it doesn't matter that much.